recipes/secapr/meta.yaml

```diff
@@ -22,27 +25,28 @@ requirements:
     - setuptools
     - biopython
     - pandas
+    - r
 
   run:
     - python
-    - biopython
-    - emboss
-    - bcftools
-    - samtools
-    - picard
-    - seqtk
-    - bwa
-    - lastz
-    - mafft
-    - muscle
-    - cogent
-    - trimmomatic
-    - trinity
-    - abyss
-    - fastqc
-    - pandas
-    - numpy 
-
+    - biopython ==1.68
+    - emboss ==6.5.7
+    - bcftools ==1.8
+    - samtools ==0.1.19
+    - picard ==1.126
+    - seqtk >=1.0.82,<=1.2
+    - bwa >=0.7
+    - lastz ==1.0.2
+    - mafft >=7.2
+    - muscle ==3.8.31
+    - trimmomatic ==0.33
+    - abyss ==1.5.2
+    - fastqc >=0.11,<=0.12
```

Comment:
You can express this as `fastqc 0.11*`

Reply:
Yes, in the general case this should even be either `fastqc 0.11.*` or `fastqc >=0.11,<0.12.0a0` -- if (hypothetically) there where an alpha version `0.12.a1`, that one would be considered `<0.12`.